Extern/IBuffer.h

```diff
@@ -15,37 +15,30 @@ See the License for the specific language governing permissions and
 limitations under the License.
 </License>
 */
```

Comment:
I don't know why GitHub decided to compare IBuffer.h with DeviceInterface.h. It should have been compared with BufferInterface.h